### DISCUSSION TITLE
Cannot set property message of [object DOMException] which has only a getter

### DISCUSSION THREAD

**Discussion by vaclavGabriel** [cypress-io/cypress#18698] [Questions and Help]
Hello everyone,
I am using Cypress Testing Library. When I try to get an element by property role (`findByRole`), the following error occurs. Could you please advise what might be the problem?

`TypeError: Cannot set property message of [object DOMException] which has only a getter`

![image](https://user-images.githubusercontent.com/48252269/139412015-1f56b345-dff3-433a-9636-01dc055d24fb.png)

**Comment by MAfzalKhan1997** [comment]
any update on this ?

**Comment by nilesh-powar-InfoTrack** [comment]
Have you used cy.xpath() anywhere in your project recently?
I did and was causing this error for me
Then i removed it and got rid of the issue

**Comment by allblueee** [comment]
any update on this ?

**Comment by jjhelguero** [comment]
This might gain more  visibility and a quicker response on the [cypress-testing-library repo](https://github.com/testing-library/cypress-testing-library) since it's a 3rd party.

**Comment by tobiasweibel** [comment]
Not resolved, but "thread" continues in https://github.com/testing-library/cypress-testing-library/issues/206
